From: Seungmin Kim <8457324+ehfd@users.noreply.github.com>
Date: Wed, 20 Mar 2025 09:00:00 +0900
Subject: [PATCH] libiconv
---
--- pulseaudio-orig/meson.build 2025-03-19 15:26:49.701695489 +0000
+++ pulseaudio/meson.build      2025-03-19 15:27:45.423932646 +0000
@@ -474,15 +474,15 @@
 endif
 
 have_iconv = false
-if cc.has_function('iconv_open')
-  iconv_dep = dependency('', required : false)
-  have_iconv = true
-  # tell the libiconv header to pretend to be libc iconv
-  cdata.set('LIBICONV_PLUG', 1)
-else
-  iconv_dep = cc.find_library('iconv', required : false)
-  have_iconv = iconv_dep.found()
-endif
+#if cc.has_function('iconv_open')
+#  iconv_dep = dependency('', required : false)
+#  have_iconv = true
+#  # tell the libiconv header to pretend to be libc iconv
+#  cdata.set('LIBICONV_PLUG', 1)
+#else
+iconv_dep = cc.find_library('iconv', required : false)
+have_iconv = iconv_dep.found()
+#endif
 if have_iconv
   cdata.set('HAVE_ICONV', 1)
   iconvconsttest = '''#include <iconv.h>
